docs/setup-and-installing.md

```diff
@@ -102,36 +128,45 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
 
 <%(/Expandable)%>
 
-<%(Callout type="alert" message="
-* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/>
-* You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
-* You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
-* Unity gives you build support for your development machine (Windows or Mac) by default.")%>
-
-<%(#Expandable title="Having problems?")%>
-If you need help using the GDK, please come and talk to us about the software and the documentation via:
+#### Make sure you have chosen the right build support
 
-**The SpatialOS forums**
+- You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment.<br/>
+- You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
+- You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
+- Unity gives you build support for your development machine (Windows or Mac) by default.
 
-Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+#### Android and iOS support
 
-**Discord**
+Mobile support is in pre-alpha. If you are developing a game for Android or iOS, refer to our GDK for Unity [mobile support documentation]({{urlRoot}}/content//mobile/overview)
```

Comment:
I wonder if we should also add the warning to the part of the get started guide where we actually build the workers. As that is the part where people might see errors that they did not expect.

Reply:
We absolutely should. I just totally forgot to do it. Adding now.

Reply:
let me know when you added it. We would like to get that PR in today for the release

Reply:
Done